payjoin/src/receive/v1/mod.rs

```diff
@@ -516,6 +520,7 @@ impl WantsInputs {
             payjoin_psbt,
             params: self.params,
             change_vout: self.change_vout,
+            receiver_inputs: inputs,
```

Comment:
I would do the collect() here and leave L499 unchanged

Reply:
Yep, 499 consumes it...so I had to collect is as Vec<> (at 494) so I could use it at 499 and 523.